data/tlg0018/tlg001/tlg0018.tlg001.1st1k-eng1.xml

```diff
@@ -0,0 +1,1060 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Works of Philo Judaeus</title>
```

Comment:
lang not needed here

Reply:
is there any reason not to keep it?

Reply:
the reason is because the header is labeled "eng" so that attribute is the default assumption for all other subelements, which is why we specify language such as" "lat" "grc" etc.

Reply:
Yes, superfluous encoding.

Reply:
It turns out I put this in specially because we are fussy about having lang attributes. Is is that English is the default? Or do we sometimes need to mark it as English?